pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -58,6 +59,7 @@ type StackInstaller interface {
 	Install(ctx context.Context, bundle releasev1alpha1.TinkerbellBundle, tinkerbellIP, kubeconfig, hookOverride string, opts ...InstallOption) error
 	UninstallLocal(ctx context.Context) error
 	Upgrade(_ context.Context, _ releasev1alpha1.TinkerbellBundle, tinkerbellIP, kubeconfig string) error
+	UpdateStackInstallerNoProxy(IP string)
```

Comment:
Let's call it this instead
```suggestion
	AddNoProxyIP(ip string)
```